blog/20250729-simplex-chat-v6-4-1-welcome-contacts-protect-groups-app-security.md

```diff
@@ -11,17 +11,111 @@ permalink: "/blog/20250729-simplex-chat-v6-4-1-welcome-contacts-protect-groups-a
 
 # SimpleX Chat v6.4.1: welcome your contacts, review members to protect groups, and more.
 
-**Will be published:** Jul 29, 2025
-
-This is a placeholder for upcoming v6.4.1 release announcement.
+**Published:** Jul 29, 2025
 
 **What's new in v6.4.1**:
 
-TODO
+- [welcome your contacts](#welcome-your-contacts) with your bio and welcome message.
```

Comment:
welcome your contacts with bio sounds strange, why not this:
```suggestion
- [welcome your contacts](#welcome-your-contacts): set profile bio and welcome message.
```
as in what's new